### PR TITLE
Feature/io 571 refactor styles colors dimens localization

### PR DESCRIPTION
**Pull request by nikolay-ahramovich-idf**

### CODE REVIEW

**Review by ekaterina-demeshko-idf — APPROVED** [review]

File: packages/presentation/lib/l10n/app_pt.arb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/l10n/app_pt.arb:1]: nice language)

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 2]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
-                              padding: EdgeInsets.zero,
-                              constraints: const BoxConstraints(),
-                              onPressed: null,
-                              icon: Image.asset(
-                                AssetsImagesPaths.respondArrow,
-                                height:
-                                    MovieDetailsScreenSizes.navigationIconSize,
-                              ),
+                          ),
+                          IconButton(
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: if u are not handling click, maybe u need just the Image/Icon widget, not IconButton?

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 3]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                            height: AppSizes.size57,
+                            decoration: const BoxDecoration(
+                              color: AppColors.transparentWhite50,
+                              shape: BoxShape.circle,
                             ),
-                            const SizedBox(
-                              height: AppSizes.size32,
+                            child: Image.asset(
+                              AssetsImagesPaths.playButtonPath,
                             ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart:137]: discussed this moment - if there's one param and we can place params in one line without crossing the vertical line - put param in one line

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 4]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size16,
+                          ),
+                          ImageWidget(
+                            moviePosterImageUrl,
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size32,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 5]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                          Text(
+                            movieDetails.title ?? '',
+                            style: MovieDetailsScreenStyles.movieNameStyle,
+                            textAlign: TextAlign.center,
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size14,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 6]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                          Text(
+                            '$runtime | ${movieDetails.certification}',
+                            style: MovieDetailsScreenStyles
+                                .movieAdditionalDataStyle,
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size9,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 7]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                          Text(
+                            getGenresPresentation(movieDetails.genres ?? []),
+                            style: MovieDetailsScreenStyles
+                                .movieAdditionalDataStyle,
+                            textAlign: TextAlign.center,
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size20,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 8]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                                RatingWidgetConfig.minCurrentRating,
+                            maxCurrentRating:
+                                RatingWidgetConfig.maxCurrentRating,
+                            starColor: AppColors.gold,
+                            starSize: MovieDetailsScreenSizes.ratingStarSize,
+                            mode: Mode.full,
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size20,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 9]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
-                                        child: Text(
-                                          'Showtime',
-                                          style: SelectionButtonStyles
-                                              .inactiveButtonTextStyle,
+                                ),
+                                Expanded(
+                                  child: Padding(
+                                    padding: const EdgeInsets.all(
+                                      AppSizes.size4,
+                                    ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart:217]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 10]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
-                                  ],
-                                )
                               ],
                             ),
-                          ],
-                        ),
+                          ),
+                          const SizedBox(
+                            height: AppSizes.size32,
+                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 11]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              Text(
+                                appLocalizations.synopsisHeaderLabel,
+                                style: MovieDetailsScreenStyles
+                                    .movieDescriptionHeaderStyle,
+                              ),
+                              const SizedBox(
+                                height: AppSizes.size14,
+                              ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 12]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                                trimExpandedText:
+                                    ' ${appLocalizations.showLessButtonLabel}',
+                                moreStyle:
+                                    MovieDetailsScreenStyles.showMoreStyle,
+                                lessStyle:
+                                    MovieDetailsScreenStyles.showMoreStyle,
+                              ),
+                              const SizedBox(
+                                height: AppSizes.size32,
+                              ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 13]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                                  Text(
+                                    appLocalizations.viewAllButtonLabel,
+                                    style:
+                                        MovieDetailsScreenStyles.showMoreStyle,
+                                  ),
+                                ],
+                              ),
+                              const SizedBox(
+                                height: AppSizes.size24,
+                              ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/screens/movie_details/movie_details_screen.dart [hunk 14]
```diff
@@ -43,391 +47,394 @@ class _MovieDetailsScreenState
...
+                                            children: [
+                                              _getThreeDotsWidget(),
+                                              const SizedBox(
+                                                height: AppSizes.size18,
+                                              )
+                                            ],
+                                          ),
+                                          const SizedBox(
+                                            width: AppSizes.size24,
+                                          ),
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/screens/movie_details/movie_details_screen.dart]: -//-

File: packages/presentation/lib/utils/dimensions.dart [hunk 15]
```diff
@@ -0,0 +1,50 @@
+class AppSizes {
+  AppSizes._();
+  static const double size0 = 0;
+  static const double size0point5 = 0.5;
```
Comment (sergei-mikhailovskii-idf) [inline packages/presentation/lib/utils/dimensions.dart]: try to avoid fractional sizes. use 0 or 1. half of the pixel/unit is not notable at UI

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

**Review by andrei-lastochkin-idf — APPROVED** [review]

**Review by ekaterina-demeshko-idf — APPROVED** [review]

**Pull request merged**